frontend/src/common/queries/collections.ts

```diff
@@ -39,3 +39,28 @@ async function fetchCollection(_: unknown, id: string): Promise<Collection> {
 export function useCollection(id: string) {
   return useQuery<Collection>([USE_COLLECTION, id], fetchCollection);
 }
+
+export async function createCollection(): Promise<string> {
+  // DEBUG
+  // DEBUG
+  // DEBUG
+  await new Promise((resolve) => {
+    setTimeout(() => {
+      resolve();
+    }, 2 * 1000);
+  });
+
+  return "673637cf-dcb7-45e1-bb88-72a27c50c8ca";
+}
+
+export const formDataToObject = function (formData: FormData) {
+  const payload: { [key: string]: unknown } = {};
```

Comment:
Is the main difference between the `unknown` and `any` the heightened restrictions on unknown?

Reply:
Great question! So `any` is looser than `unknown` in the sense that when an `any` value is assigned to another typed value like `string`, it'll pass type checking, but if it was an `unknown`, then passing that value to a `string` type will not pass type checking!

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-0.html#new-unknown-top-type